nim/assetlist.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "nim",
+    "assets": [
+        {
+            "description": "The native token of Nim Network.",
+            "extended_description": "Nim Network is a highly-adoptable AI Gaming chain that will provide the ultimate ecosystem for exploration and development of games at the intersection of Web3 and AI.",
+            "denom_units": [
+                {
+                    "denom": "anim",
+                    "exponent": 0
+                },
+                {
+                    "denom": "nim",
+                    "exponent": 18
+                }
+            ],
+            "type_asset": "sdk.coin",
+            "base": "anim",
+            "name": "NIM Network",
+            "display": "nim",
+            "symbol": "NIM",
+            "images": [
+                {
+                    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dymension/images/nim.png",
```

Comment:
this should be in nim's images directory

Reply:
bad copy/paste typo, fixed. images are also added there.